backend/src/v4/models/jsonAssets.js

```diff
@@ -278,8 +277,6 @@ const addSuperMeshMappingsToStream = async (account, model, revId, jsonFiles, le
 			cacheStream.emit("error", err);
 		});
 
```

Comment:
@FJThiel the linter is complaining about this line! You can run the linter manually before committing if VSCode is not set to use it automatically. ('yarn lint' I think).